### PR TITLE
Closes #2124 Translate the adapter readme to Spanish

### LINKED ISSUE #2124: Translate the adapter readme to Spanish
I'll translate the readme file of the adapter design pattern to Spanish and place it under localization.es.adapter.

### PR DESCRIPTION
**Pull request by aitorfi**
# Translate the adapter readme to Spanish
Translates the adapter design pattern's readme file to Spanish. Closes #2124

### CODE REVIEW

File: localization/es/adapter/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+---
+layout: pattern
+title: Adapter
+folder: adapter
+permalink: /patterns/adapter/
+categories: Structural
+language: es
+tags:
+ - Gang of Four
+---
```
Comment (iluwatar) [inline localization/es/adapter/README.md:10]: Please check the new yaml frontmatter requirements at https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

**Review by iluwatar — CHANGES_REQUESTED** [review]

**Review by iluwatar — APPROVED** [review]

**Comment by iluwatar** [comment]
Thanks @aitorfi, well done!

@all-contributors please add @aitorfi for translation

**Comment by KarmaTashiCat** [comment]
Hello, if is needed contribution to spanish translation I can do one or more. @iluwatar If it is please tell me from wich file I should start for not make the same as @aitorfi would working. Next pattern readme.md would be aggregator-microservice. I can start with it, or next or 3d or 10th next, the last ... tell me and I will start with one and followings.

**Comment by aitorfi** [comment]
Hello @KarmaTashiCat, I'm not currently working on any translations so pick the design pattern you want. You could create an issue for each translation and that way everyone will know what you are working on. Also, for future conversations on Spanish translations it would be better to use the issue #2277 instead of an already merged PR.

**Pull request merged**